pydomcfg/domzgr/zco.py

```diff
@@ -59,216 +43,268 @@ def __call__(
             Minimum thickness for the top layer (units: m)
         pphmax: float
             Depth of the last W-level (total depth of the ocean basin) (units: m)
-        ppacr: int
-            Stretching factor, nondimensional > 0. The larger ppacr,
-            the smaller the stretching. Values from 3 to 10 are usual.
-            (default = 0., no stretching, uniform grid)
-        ppkth: float
+        ppkth: float, default = 0
             Model level at which approximately max stretching occurs.
-            Nondimensional > 0, usually of order 1/2 or 2/3 of jpk.
-            (default = 0., i.e. no stretching, uniform grid)
-        ppsur: float, optional
-            Coeff. controlling distibution of vertical levels
-            (default = None, i.e. computed from ppdzmin, pphmax, ppkth and ppacr)
-        ppa0: float, optional
-            Coeff. controlling distibution of vertical levels
-            (default = None, i.e. computed from ppdzmin, pphmax, ppkth and ppacr)
-        ppa1: float, optional
-            Coeff. controlling distibution of vertical levels
-            (default = None, i.e. computed from ppdzmin, pphmax, ppkth and ppacr)
-        ldbletanh: bool
-            Logical flag to use or not double tanh stretching function
-            (default = False)
-        ppa2: float
-            Double tanh stretching function parameter
-        ppkth2: float
-            Double tanh stretching function parameter
-        ppacr2: float
-            Double tanh stretching function parameter
-        ln_e3_dep: bool
-            Logical flag to comp. e3 as fin. diff. (True) or
-            analyt. (False) (default = True)
+            Nondimensional > 0, usually of order 1/2 or 2/3 of ``jpk``.
+            (0 means no stretching, uniform grid)
+        ppacr: int, default = 0
+            Stretching factor, nondimensional > 0. The larger ``ppacr``,
+            the smaller the stretching. Values from 3 to 10 are usual.
+            (0 means no stretching, uniform grid)
+        ppsur, ppa0, ppa1: float, optional
+            Coefficients controlling distibution of vertical levels
+            (None: compute from ``ppdzmin``, ``pphmax``, ``ppkth`` and ``ppacr``)
+        ppa2, ppkth2, ppacr2: float, optional
+            Double tanh stretching function parameters.
+            (None: Double tanh OFF)
+        ldbletanh: bool, optional
+            Logical flag to switch ON/OFF the double tanh stretching function.
+            This flag is only needed for compatibility with NEMO DOMAINcfg tools.
+            Just set ``ppa2``, ``ppkth2``, and ``ppacr2`` to switch ON
+            the double tanh stretching function.
```

Comment:
I think this flag is redundant. I suggest we keep it just for backward compatibility with NEMO tools.